mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -556,10 +561,9 @@ class Completions(
       ) // !sym.info.typeSymbol.is(Flags.Method) does not detect Java methods
 
     private def isNotClassOrTraitOrTheyAreValidForPos(sym: Symbol) =
-      cursorPositionCondition.isClassOrTraitValidForPos || sym.is(
-        Flags.Method
-      ) || (!sym.isClass && !sym.info.typeSymbol
-        .is(Trait))
+      cursorPositionCondition.isClassOrTraitValidForPos ||
+        sym.is(Flags.Method) || sym.is(Package) ||
```

Comment:
I would say it is better to put `|| sym.is(Package)` in `include` because package is really valid for all positions of type completion I can think of:

- `val a : scala.colle@@`
- `new scala.collec@@`
- `scala.collec@@`

Reply:
Good catch, I added the test case and fixed the issues that popped up.